src/index.ts

```diff
@@ -182,3 +184,12 @@ export {
  * Epics implementations
  */
 export {sendOperationEpicImpl} from './epics/view'
+
+/**
+ * Autosave middleware utils
+ */
+export {
+    autosaveRoutine,
+    bcHasPendingAutosaveChanges,
```

Comment:
I do not think we should export `bcHasPendingAutosaveChanges` and `checkUnsavedChangesOfBc` as their are trivial to implement client-side.

Reply:
Ok